lib/tokens.js

```diff
@@ -433,8 +439,14 @@ async function _parseToken({token}) {
   // "payload" and optional "meta"
   const parsed = {};
   try {
-    const {type, payload, meta} = await cborldDecode(
-      {appContextMap, cborldBytes: token, documentLoader});
+    const {type, payload, meta} = await cborldDecode({
+      appContextMap,
+      cborldBytes: token,
+      documentLoader: async url => {
+        const {document: CIT_CONTEXT} = await documentLoader(url);
```

Comment:
Can we DRY this up a bit by making a `_documentLoader` helper and using it here and on L421?

Reply:
Of course, will do.

Reply:
Fixed.